crates/autopilot/src/run_loop.rs

```diff
@@ -582,7 +583,9 @@ impl RunLoop {
                     .solution()
                     .orders()
                     .iter()
-                    .flat_map(|(_, order)| [order.sell.token, order.buy.token])
+                    .flat_map(|(_, order)| {
+                        [order.sell.token.wrap(weth), order.buy.token.wrap(weth)]
```

Comment:
Fine to merge as is since this function already exists like this in the driver but am I the only one who feels like `.wrap()` wasn't a good name to begin with?
It feels a bit weird to have this as a core mechanism and not have a good name for it. 🤔 
Maybe `.or_wrapped(weth)` or `.as_erc20(weth)`? 🤷‍♂️

Reply:
I would agree. Switched to `as_erc20`